slack_connector/main.py

```diff
@@ -96,7 +138,7 @@ def main():
     if args.channel_id == 'secret':
         args.channel_id = os.environ.get('SLACK_CHANNELID')
 
```

Comment:
This is optional for now. But I recommend adding checks on all the  os.env vars. 

Alternatively.  You could add test cases to catch Exceptions raised by os.environ.get